crates/oxc_linter/src/snapshots/no_new_statics.snap

```diff
@@ -0,0 +1,46 @@
+---
+source: crates/oxc_linter/src/tester.rs
+---
+  ⚠ eslint-plugin-promise(no-new-statics): Disallow calling `new` on a Promise static method
+   ╭─[no_new_statics.tsx:1:1]
+ 1 │ new Promise.resolve()
```

Comment:
Nit: do we need to shrink to point on `new`?

Reply:
Why not, it seems to make sense and was easy to do :)